tests/ui_tests/app/page/common/base_page.py

```diff
@@ -501,9 +501,10 @@ def get_job_rows_from_popup(self) -> List[WebElement]:
         return self.find_elements(AuthorizedHeaderLocators.JobPopup.job_row)
 
     def click_on_task_row_by_name(self, task_name: str):
-        for task in self.get_job_rows_from_popup():
-            if task.text == task_name:
-                task.click()
+        for task in self.find_elements(AuthorizedHeaderLocators.JobPopup.job_row):
+            name = self.find_child(task, AuthorizedHeaderLocators.JobPopup.JobRow.job_name)
+            if name.text == task_name:
+                name.click()
```

Comment:
raise exception if requested task not found

Reply:
added